src/modules/client/components/ClientAlerts.tsx

```diff
@@ -0,0 +1,116 @@
+import { Alert, AlertTitle, Box, Typography } from '@mui/material';
+import { Stack } from '@mui/system';
+import TitleCard from '@/components/elements/TitleCard';
+import theme from '@/config/theme';
+import {
+  clientNameAllParts,
+  parseAndFormatDateTime,
+} from '@/modules/hmis/hmisUtil';
+import {
+  ClientAlertFieldsFragment,
+  ClientFieldsFragment,
+} from '@/types/gqlTypes';
+
+interface ClientAlertCardProps {
+  alert: ClientAlertFieldsFragment;
+  client: ClientFieldsFragment;
+  shouldShowClientName?: boolean;
+}
+
+const ClientAlertCard: React.FC<ClientAlertCardProps> = ({
+  alert,
+  client,
+  shouldShowClientName = false,
+}) => {
+  const priority = alert.severity || 'low';
+
+  const priorityColors: { [index: string]: any } = {
+    high: {
+      header: theme.palette.alerts.highPriorityErrorBackground,
+      body: `${theme.palette.error.main}04`,
+      border: `${theme.palette.error.main}50`,
+    },
+    medium: {
+      header: theme.palette.alerts.mediumPriorityHeaderBackground,
+      body: theme.palette.alerts.mediumPriorityBodyBackground,
+      border: theme.palette.alerts.mediumPriorityBorder,
+    },
+    low: {
+      header: theme.palette.alerts.lowPriorityHeaderBackground,
+      body: theme.palette.alerts.lowPriorityBodyBackground,
+      border: theme.palette.alerts.lowPriorityHeaderBackground,
+    },
+  };
+
+  return (
+    <Alert
+      sx={{
```

Comment:
Here's a screenshot of what this looks like (high priority version)
<img width="1512" alt="Screenshot 2024-01-25 at 2 34 11 PM" src="https://github.com/greenriver/hmis-frontend/assets/16002775/77b135ec-267a-41be-b1e7-e7cf8a7165b1">